rootfs/api/models/config.py

```diff
@@ -105,6 +111,40 @@ def set_tags(self, previous_config):
 
         raise DeisException(message)
 
+    def set_healthcheck(self, previous_config):
+        data = getattr(previous_config, 'healthcheck', {}).copy()
+        new_data = getattr(self, 'healthcheck', {}).copy()
+        # update the config data for healthcheck if they are not
+        # present for per proctype
+        if 'livenessProbe' in data.keys() or 'readinessProbe' in data.keys():
```

Comment:
this code gets called only during the load from previous controller release data,i can't insert data as serializers will fail.
same for below.